tool/grind.dart

```diff
@@ -177,19 +177,20 @@ void npmRelease() {
       .writeAsStringSync((const JsonEncoder.withIndent('    ')).convert(json));
 
   copy(new File(p.join(sourceDir, 'index.js')), dir);
-  copy(new File(p.join(sourceDir, 'README.md')), dir);
   copy(new File('ISSUES.md'), dir);
   copy(new File('LICENSE'), dir);
   copy(new File('3RD_PARTY'), dir);
   copy(new File(p.join('docs', 'validation.schema.json')), dir);
+
+  log("Building npm README...");
+  run(npmExecutable, arguments: ['install'], workingDirectory: 'tool/npm_template');
+  copy(new File(p.join(sourceDir, 'README.md')), dir);
+  run(npmExecutable, arguments: ['run', 'docs'], workingDirectory: 'tool/npm_template');
```

Comment:
Could you move that section back to `npm` task (probably with other documentation files above)?
`npm-release` and `npm-debug` are meant to perform only Dart->JS compilation.